src/Command/MonitorCommand.php

```diff
@@ -123,14 +124,14 @@ private function scheduledTaskFailed(): bool
             'FroshTools.config.monitorTaskGraceTime'
         );
 
-        $date = new \DateTime();
+        $date = new \DateTime('now',new DateTimeZone('UTC'));
         $date->modify(sprintf('-%d minutes', $minutes));
 
         $criteria = new Criteria();
         $criteria->addFilter(
             new RangeFilter(
                 'nextExecutionTime',
-                ['lte' => $date->format(\DATE_ATOM)]
+                ['lte' => $date->format('Y-m-d H:i:s.v')]
```

Comment:
Would be better to use Shopware's `Defaults::STORAGE_DATE_TIME_FORMAT` constant instead.